keras_nlp/utils/text_generation.py

```diff
@@ -61,10 +68,13 @@ def greedy_search(
 
     Args:
         token_probability_fn: a callable, which takes in input_sequence
-            and output the probability distribution of the next token.
+            and output the probability distribution or the logits of the next
+            token.
         prompt: a list or a Tensor, can be 1D or 2D, the initial tokens to
             append generated tokens.
         max_length: int. The max length of generated text.
+        from_logits: bool. Indicates whether `token_probability_fn` outputs
```

Comment:
You document this but didn't actually add it? Should we just leave it off as for greedy search it is a no-op?